src/table/virtualized-table/hooks/use-measure-text.ts

```diff
@@ -22,8 +24,29 @@ export default function useMeasureText(
     return {
       measureText: (text) => memoizedMeasureText(text).width,
       estimateWidth: (length: number) => memoizedMeasureText(MAGIC_DEFAULT_CHAR).width * length,
+      estimateLineCount: (text: string, maxWidth: number) => {
```

Comment:
this is the one you would optimize later right?

Reply:
Yes